app/Helpers/database/achievement-creator.php

```diff
@@ -159,10 +159,9 @@ function attributeDevelopmentAuthor(string $author, int $count, int $points): vo
     $oldContribCount = $user->ContribCount;
     $oldContribYield = $user->ContribYield;
 
-    $user->ContribCount += $count;
-    $user->ContribYield += $points;
-    $user->timestamps = false;
-    $user->save();
+    // use raw statement to perform atomic update
+    legacyDbStatement("UPDATE UserAccounts SET ContribCount = ContribCount + $count," .
```

Comment:
Now that we have reduced the DB connection to just one and those functions using the exact same signature we could deprecate/move away from the legacy* helpers and use the features directly.
```php
// used to have the connection
// DB::connection('mysql_legacy')->statement($query, $bindings);
// now it's just the same
DB::statement($query, $bindings);
```
I'll prepare a separate PR for that.